### PR TITLE
finished the ui screen for the home screen

### PR DESCRIPTION
**Pull request by Momen-Code**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,7 +3,18 @@ import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
...
+import {
+  HomePage,
+  Login,
+  SignUp,
+  StudentPage,
+  TeacherPage,
+} from "platform/routes";
+
+//Teacher Routes
+import { Home } from "teacher/routes";
```
Comment (buildwithashmawy) [inline src/App.tsx]: try to rename every imported component with a prefix of its module name, like: PlatformHome | TeacherHome | TeacherDashboardHome

File: src/assets/styles/_global.scss [hunk 2]
```diff
@@ -171,6 +172,17 @@ button {
 }
 /**********************************/
 
+@mixin gradientColor($direction) {
+  @if $direction == right {
```
Comment (buildwithashmawy) [inline src/assets/styles/_global.scss]: Its better to use direction as degree's instead of just passing the direction as literal

**Review by buildwithashmawy — COMMENTED** [review]
Please take a look at the comments

**Review by buildwithashmawy — CHANGES_REQUESTED** [review]
ASAP

File: src/App.tsx (hunk 1 above)
Comment (Momen-Code) [reply src/App.tsx]: Done

File: src/assets/styles/_global.scss (hunk 2 above)
Comment (Momen-Code) [reply src/assets/styles/_global.scss]: Done

**Review by buildwithashmawy — APPROVED** [review]
👍

**Pull request merged**